### PR TITLE
AWS backup changes

### PR DESCRIPTION
**Pull request by chrisahl**
For issue https://issues.redhat.com/browse/AAP-13159

### CODE REVIEW

**Review by itdove — APPROVED** [review]

**Review by tphee — APPROVED** [review]

File: stories/topics/proc-aws-deleting-backups-playbook.adoc [hunk 1]
```diff
@@ -49,13 +49,14 @@ aws_backups_delete:
     delete:
 ----
 
-With `aws_backup_names` such as "backup1" and "backup2".
+The `aws_backup_names` parameter must specify an array of strings, for example, `["backup1","backup2"]`.
+The `delete` parameter must be set to `true` to successfully delete.
 
-. To delete the backups, run the command generator to generate the deletion backup command.
+. To delete the backups, run the command generator to generate the aws_backups_delete command.
```
Comment (ianf77) [inline stories/topics/proc-aws-deleting-backups-playbook.adoc:55]: wrap aws_backup_delete in backticks

File: stories/topics/proc-aws-generate-restore-data-file.adoc [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 = Generating the restore data file
 
-The following commands create a directory, and populate it with an empty data template that, when completed, will be used during the backup. 
+The following commands create a directory, and populate it with an empty data template that, when completed, will be used during the restore.
```
Comment (ianf77) [inline stories/topics/proc-aws-generate-restore-data-file.adoc:5]: data template that, when completed, is used during...

File: stories/topics/proc-aws-run-restore-playbook.adoc [hunk 3]
```diff
@@ -36,4 +36,4 @@ aws_s3_bucket=ansible-automation-platform-bucket aws_ssm_bucket_name=aap-ssm-buc
 
 ----
 . The playbook can take some time to create a restored deployment.
-After you have run the playbook successfully, a restored deployment is available in AWS CloudFormation matching the name provided for it in the `extra_vars.yml` file.
+After you have run the playbook successfully, a restored deployment is available in AWS CloudFormation matching the name provided for it in the `restore.yml` file.
```
Comment (ianf77) [inline stories/topics/proc-aws-run-restore-playbook.adoc:39]: When you have run

File: stories/topics/proc-gcp-deleting-backups-playbook.adoc [hunk 4]
```diff
@@ -53,7 +53,7 @@ docker run --rm --env PLATFORM=GCP -v </path/to/gcp/service-account.json>:/home/
 -e 'gcp_service_account_credentials_json_path=/home/runner/.gcp/credentials  gcp_bucket_backup_name=<bucket> \
 backup_names=<backup_names> delete=True'
 ----
-. Run the supplied backup command to trigger the backup.
+. Run the supplied backup command to trigger the deletion of the backups.
```
Comment (ianf77) [inline stories/topics/proc-gcp-deleting-backups-playbook.adoc:56]: to delete the backups.

**Review by ianf77 — COMMENTED** [review]
A few comments

File: stories/topics/proc-aws-deleting-backups-playbook.adoc (hunk 1 above)
Comment (chrisahl) [reply stories/topics/proc-aws-deleting-backups-playbook.adoc:55]: fixed

File: stories/topics/proc-aws-deleting-backups-playbook.adoc [hunk 5]
```diff
@@ -66,10 +67,10 @@ Command to run playbook:
 
 docker run --rm --env PLATFORM=AWS -v ~/.aws/credentials:/home/runner/.aws/credentials:ro \
 --env ANSIBLE_CONFIG=../aws-ansible.cfg  $IMAGE redhat.ansible_on_clouds.aws_backups_delete \
--e 'aws_region=<region> aws_s3_bucket=<bucket> aws_backup_names=["mybackup"] delete=True'
+-e 'aws_region=<region> aws_s3_bucket=<bucket> aws_backup_names=["backup1","backup2"] delete=True'
 ===============================================
 ----
-. Run the supplied backup command to trigger the backup deletion.
+. Run the supplied backup command to trigger the deletion of the backups.
```
[the remark this answers is not in the document]
Comment (chrisahl) [reply stories/topics/proc-aws-deleting-backups-playbook.adoc:73]: fixed

File: stories/topics/proc-aws-generate-restore-data-file.adoc (hunk 2 above)
Comment (chrisahl) [reply stories/topics/proc-aws-generate-restore-data-file.adoc:5]: fixed

File: stories/topics/proc-aws-run-restore-playbook.adoc (hunk 3 above)
Comment (chrisahl) [reply stories/topics/proc-aws-run-restore-playbook.adoc:39]: fixed

File: stories/topics/proc-gcp-deleting-backups-playbook.adoc [hunk 6]
```diff
@@ -34,7 +34,7 @@ gcp_backups_delete:
 The `backup_names` parameter must specify an array of strings, for example, `["backup1","backup2"]`.
 The `delete` parameter must be set to `true` to successfully delete.
 
-. To run the delete backups, run the command generator to generate the backups_delete command.
+. To delete the backups, run the command generator to generate the gcp_backups_delete command.
```
[the remark this answers is not in the document]
Comment (chrisahl) [reply stories/topics/proc-gcp-deleting-backups-playbook.adoc:37]: fixed

File: stories/topics/proc-gcp-deleting-backups-playbook.adoc (hunk 4 above)
Comment (chrisahl) [reply stories/topics/proc-gcp-deleting-backups-playbook.adoc:56]: fixed

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**